charts/uptime-kuma/values.yaml

```diff
@@ -106,6 +106,13 @@ affinity: {}
 
 serviceMonitor:
   enabled: false
+  authentication:
+    enabled: true
```

Comment:
only question I have is, why you default it to _enabled: true_

personally, would default it to false unless uptime-kuma requires it to expose the metrics, but in that case the toggle would be obsolete anyway.

Reply:
Could do that, but I wanted to make this non-breaking. And since authentication was enabled beforehand, I kept it. I can also "break" it and disable it by default. Would make more sense from my perspective, too.